From 8f2b3cd7a80621f3fb5a02a5cd53f0b132e44ebc Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 15 Sep 2014 10:52:32 +0200
Subject: [PATCH 06/60] net/cert: increase default key length for
 newly-generated RSA keys

Should we add something for chrome://settings, or just leave
it as hidden as it was before?
---
 net/cert/x509_util.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/cert/x509_util.cc b/net/cert/x509_util.cc
index a5d583d0eb1b..ab84746791c1 100644
--- a/net/cert/x509_util.cc
+++ b/net/cert/x509_util.cc
@@ -54,7 +54,7 @@ bool DecodeTime(const der::GeneralizedTime& generalized_time,
 namespace x509_util {
 
 // RSA keys created by CreateKeyAndSelfSignedCert will be of this length.
-static const uint16_t kRSAKeyLength = 1024;
+static const uint16_t kRSAKeyLength = 2048;
 
 // Certificates made by CreateKeyAndSelfSignedCert and
 //  CreateKeyAndChannelIDEC will be signed using this digest algorithm.
-- 
2.12.2

